### PR TITLE
Add tutorial files

### PR DESCRIPTION
**Pull request by mint-thompson**
Tutorial files that were previously kept in another repository are added locally. The fsh-tutorial directory contains these files.

Add step to the github action for deploying the site that rebuilds the tutorial zip.

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -13,6 +13,9 @@ jobs:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Create Tutorial Zip
+        run: zip content/docs/SUSHI/tutorial/fsh-tutorial.zip fsh-tutorial/* -r -FS
+
```
Comment (jafeltra) [inline .github/workflows/deploy.yaml:18]: I might be overlooking something, but if we have the zip file added in the source code, do we need to re-create the zip file on deployment? If this is to ensure it always stay up to date, do we need a version check in or can we just gitignore it?

**Review by jafeltra — COMMENTED** [review]
This looks good! I just had one question that I left inline.

**Review by jafeltra — APPROVED** [review]
This looks good to me! I ran the `zip-tutorial` script and it worked well on my computer. When I run it, git does show it as changed, but I think that's fine. I also confirmed that the zip that's checked in works on my computer, and it seems to work without any issues.

So I think this is good to go! Thanks for adding the script!

**Review by cmoesel — APPROVED** [review]
Looks good!

**Pull request merged**